R/tags.R

```diff
@@ -492,6 +496,8 @@ tagWrite <- function(tag, textWriter, indent=0, eol = "\n") {
 
   # write attributes
   for (attrib in names(attribs)) {
+    # Can not display attrib without a key
+    if (identical(attrib, "")) next
```

Comment:
Oh, I see, so this would 'fix' this situation?

```r
x <- div()
x$attribs[[1]] <- "a"
```

Reply:
Part of me thinks that we should also `stop()` here with an informative error

Reply:
Correct. No error is thrown.

Tested below